app/views/catalog/_search_results.html.erb

```diff
@@ -9,18 +8,20 @@
     <% @page_title = t 'blacklight.search.masthead_title', title: page_location.access_point.to_s.gsub(/_/, ' ').capitalize.pluralize, application_name: "#{I18n.t('blacklight.application_name')} catalog" %>
   <% end %>
 <% else %>
-  <% @page_title = t('blacklight.search.page_title.title', constraints: render_search_to_page_title(search_state), application_name: application_name) %>
+  <% constraints_text = render_search_to_page_title(search_state) %>
+  <% if constraints_text.present? %>
+    <% @page_title = t('blacklight.search.page_title.title', constraints: constraints_text, application_name: application_name) %>
+  <% else %>
+    <% @page_title = "#{application_name} catalog" %>
+  <% end %>
 <% end %>
-
 <% view_config = blacklight_config.view_config(document_index_view_type, action_name: action_name) %>
 <% content_for(:head) do -%>
   <%= render '/catalog/opensearch_response_metadata' %>
   <%= rss_feed_link_tag %>
   <%= atom_feed_link_tag %>
   <%= json_api_link_tag %>
 <% end -%>
-
-
```

Comment:
Why is this removing all the whitespace? I think this helps readability.

Reply:
I fixed this.